src/referentielijsten/templates/admin/base_site.html

```diff
@@ -48,6 +48,13 @@ <h1 id="site-name">
 {% block nav-global %}
 {% endblock %}
 
+{% block messages %}
+    {% if settings.ENVIRONMENT_SHOWN_IN_ADMIN %}
+        <div class="env env-{{ settings.ENVIRONMENT }}">{{ settings.ENVIRONMENT }}</div>
+    {% endif %}
+    {{ block.super }}
+{% endblock %}
+
```

Comment:
Similar to this comment here: https://github.com/open-zaak/open-notificaties/pull/155#discussion_r1630819492

Add this to open-api-framework?